src/Test.FunctionalTests.BinSkim.Driver/BaselineTestsData/Expected/BinSkim.win-x64.ni.dll.sarif

```diff
@@ -556,33 +556,9 @@
             }
           ]
         },
-        {
-          "ruleId": "BA4001",
-          "ruleIndex": 23,
-          "kind": "notApplicable",
-          "level": "none",
-          "message": {
-            "id": "NotApplicable_InvalidMetadata",
-            "arguments": [
-              "BinSkim.win-x64.ni.dll",
-              "ReportPECompilerData",
-              "image is a managed IL library (i.e., ahead of time compiled) assembly"
-            ]
-          },
-          "locations": [
-            {
-              "physicalLocation": {
-                "artifactLocation": {
-                  "uri": "file:///Z:/src/Test.FunctionalTests.BinSkim.Driver/BaselineTestsData/BinSkim.win-x64.ni.dll",
-                  "index": 0
-                }
-              }
-            }
-          ]
-        },
         {
           "ruleId": "BA5001",
```

Comment:
>"ruleId": "BA5001", [](http://example.com/codeflow?start=10&length=19)

Is this supposed to be changing from `BA4001` to `BA5001`? Seeing this in a few different sarif files.